aips/aip-30.md

```diff
@@ -14,11 +14,20 @@ updated (*optional): 5/3/2023
 
 ## Summary
 
-This AIP proposes a 1.5% yearly decrease in staking rewards, which is part of the Aptos tokenomics requirement. 
+Currently, the maximum staking reward rate is a constant annualized rate of 7%. This AIP proposes a 1.5% yearly decrease in staking rewards:
```

Comment:
"1.5% yearly decrease in staking rewards" -> `1.5% yearly decrease in staking rewards rate`

Reply:
Also good to mention this is to align with tokenomics and link to the overview for tokenomics